modules/publicize/publicize-jetpack.php

```diff
@@ -156,11 +156,12 @@ function admin_page_load() {
 				$wpcom_blog_id = !empty( $wpcom_blog_id ) ? $wpcom_blog_id : $stats_options['blog_id'];
 
 				$user = wp_get_current_user();
+				$custom_site_url = apply_filters( 'publicize_custom_site_url', site_url() );
```

Comment:
Could you add some documentation before the filter, as explained [here](https://make.wordpress.org/core/handbook/inline-documentation-standards/php-documentation-standards/#4-hooks-actions-and-filters)?